libs/platform/src/lib/components/dynamic-page/dynamic-page.component.ts

```diff
@@ -69,42 +71,44 @@ export class DynamicPageComponent extends BaseComponent implements AfterContentI
     @Input()
     offset = 0;
```

Comment:
if `offset` is not being used, we should probably also remove from the @Input and add to breaking change?

Reply:
I would prefer just to add `@deprecated` for the beginning

Reply:
added `@deprocated`